src/python/impactx/dashboard/Input/space_charge_configuration/spaceChargeMain.py

```diff
@@ -0,0 +1,203 @@
+from trame.widgets import vuetify
+
+from ...trame_setup import setup_server
+from ..generalFunctions import generalFunctions
+from .spaceChargeFunctions import SpaceChargeFunctions
+
+server, state, ctrl = setup_server()
+
+# -----------------------------------------------------------------------------
+# Default
+# -----------------------------------------------------------------------------
+
+state.dynamic_size = False
+state.max_level = 3
+state.n_cell = [0.0, 0.0, 0.0]

```

Comment:
`n_cell` are _integers_.

Validation: they MUST be `>1`  if space charge (any solver) is on.

A sensible, coarse default is:
```suggestion
state.n_cell = [32, 32, 32]
```